src/app/shared/mydspace-actions/workspaceitem/workspaceitem-actions.component.ts

```diff
@@ -77,6 +100,25 @@ export class WorkspaceitemActionsComponent extends MyDSpaceActionsComponent<Work
     );
   }
 
+
+  ngOnInit(): void {
+    const activeEPerson$ = this.authService.getAuthenticatedUserFromStore();
+
+    this.isAdmin$ = activeEPerson$.pipe(
+      switchMap((user: EPerson) => this.authorizationService.isAuthorized(FeatureID.AdministratorOf, user.uuid)));
+
+    this.canEditItem$ = activeEPerson$.pipe(
```

Comment:
Just a note here on how we might improve performance... we do *not* need to even check `canEditItem` if the above call to `isAdmin` returns true.   So, this call is only necessary when isAdmin==false.  If it's easy enough to add that sort of check, then I'd recommend doing it.

Reply:
@alisaismailati :  On second thought, why are we even checking `isAdmin`??  Shouldn't we just check `canEditItem`?  If you are an Admin then that should already be "true".

We might be able to remove all the isAdmin checks altogether.

Reply:
Hello @tdonohue,
Requested changes are done, isAdmin checks are removed.